### PR TITLE
New package: python-pex-1.4.6

### PR DESCRIPTION
**Pull request by dan-santana**

### CODE REVIEW

File: srcpkgs/python-pex/template [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+post_build() {
+	cd docs
+	make man
+}
+
+post_install() {
+	vman docs/_build/man/pex.1 pex2.1
+	vlicense LICENSE
```
Comment (maxice8) [inline srcpkgs/python-pex/template]: Apache-2.0 doesn't need LICENSE

Comment (dan-santana) [reply srcpkgs/python-pex/template]: Wow, I didn't know this one. Should I remove it from `minikube`, `telepresence` and `dockeru-machine-driver-kvm2` (the packages I'm maintaining) too?

File: srcpkgs/python-pex/template [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+pkgname=python-pex
+version=1.4.6
+revision=1
+wrksrc="pex-${version}"
+noarch=yes
+build_style=python-module
+pycompile_module="pex"
+hostmakedepends="python-setuptools python3-setuptools python-Sphinx python3-Sphinx"
+depends="python python-wheel"
+short_desc="A library and tool for generating .pex (Python EXecutable) files"
```
[the remark this answers is not in the document]
Comment (dan-santana) [reply srcpkgs/python-pex/template]: fixed!

**Comment by dan-santana** [comment]
Anything missing for this package to be merged?

**Comment by Hoshpak** [comment]
looks fine to me, merging.

**Pull request merged**